### PR TITLE
Timeline Mobile View

### PR DESCRIPTION
**Pull request by yasmeen45**
## Description

Adjusted timeline to mobile view.

## Updates

Adjusted measurements to fit on mobile view and changed the look of the filters dropdown.

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [x] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by yasmeen45** [comment]
[Timeline mobile view](https://trello.com/c/4PCDsvqU/89-timeline-mobile-view)

**Comment by ElisaHu1** [comment]
@yasmeen45 can you fix the merge conflict?

**Review by ElisaHu1 — APPROVED** [review]
currently no functionality for filter and search been added to the mobile view, new ticket is needed

**Review by svolterra — COMMENTED** [review]
The UI design looks really good 😄👏  Amazing work! 💯 My one comment at the moment is that the search functionality that you added in your most recent commit (July 3) does not seem to be working on my end 🤔

**Comment by ElisaHu1** [comment]
@yasmeen45 filter and search does not work for me either

**Comment by yasmeen45** [comment]
Filtering should work now!

**Review by svolterra — APPROVED** [review]
Looks great! Amazing work Yasmeen! 👏 🎉 🎊

File: src/components/FilterDropdown/MobileAuthorsFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    setFilter({ ...filterBy, author: typeof value === 'string' ? value.split(',') : value })
+  };
+
+  return (
+
+    <FormControl sx={{ m: 1, width: '100%' }}>
+      <Select
+        labelId="demo-multiple-name-label"
+        id="demo-multiple-name"
```
Comment (svolterra) [inline src/components/FilterDropdown/MobileAuthorsFilter.tsx:46]: Leaving a note here for future reference that we probably need to rename these lol

**Pull request merged**